hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenAirdropTest.java

```diff
@@ -2270,27 +2182,21 @@ class DeleteAccount {
         @DisplayName("to fungible token pending airdrop")
         final Stream<DynamicTest> canNotDeleteAccountRelatedToAirdrop() {
             var receiver = "receiverToDelete";
-            return defaultHapiSpec("should fail - ACCOUNT_HAS_PENDING_AIRDROPS")
-                    .given()
-                    .when()
-                    .then(
-                            cryptoCreate(receiver).maxAutomaticTokenAssociations(0),
-                            tokenAirdrop(moving(10, FUNGIBLE_TOKEN).between(OWNER, receiver))
-                                    .payingWith(OWNER),
-                            cryptoDelete(OWNER).hasKnownStatus(ACCOUNT_HAS_PENDING_AIRDROPS));
+            return hapiTest(
+                    cryptoCreate(receiver).maxAutomaticTokenAssociations(0),
+                    tokenAirdrop(moving(10, FUNGIBLE_TOKEN).between(OWNER, receiver))
+                            .payingWith(OWNER),
+                    cryptoDelete(OWNER).hasKnownStatus(ACCOUNT_HAS_PENDING_AIRDROPS));
         }
 
         @HapiTest
         @DisplayName("to non-fungible token pending airdrop")
         final Stream<DynamicTest> canNotDeleteAccountRelatedToNFTAirdrop() {
-            return defaultHapiSpec("should fail - ACCOUNT_HAS_PENDING_AIRDROPS")
-                    .given()
-                    .when()
-                    .then(
-                            tokenAirdrop(TokenMovement.movingUnique(NON_FUNGIBLE_TOKEN, 10L)
-                                            .between(OWNER, RECEIVER_WITH_0_AUTO_ASSOCIATIONS))
-                                    .payingWith(OWNER),
-                            cryptoDelete(OWNER).hasKnownStatus(ACCOUNT_HAS_PENDING_AIRDROPS));
+            return hapiTest(
+                    tokenAirdrop(TokenMovement.movingUnique(NON_FUNGIBLE_TOKEN, 10L)
+                                    .between(OWNER, RECEIVER_WITH_0_AUTO_ASSOCIATIONS))
+                            .payingWith(OWNER),
```

Comment:
wait, this suite is finally over?  I was thinking it would keep going and going and going ...

Reply:
I dub thee EnergizerTest